reports/blueprint/report.tex

```diff
@@ -51,33 +51,51 @@ \section*{Executive Summary}
 \listoftables
 \newpage
 \setstretch{1.5}
-% REPORT BODY (20 Page Max)
 \section{Introduction} \label{sec:intro} % John
-% half a page ish
+% As short as possible. Total BS LMAO
 Welcome to this 1.5 spaced introduction. it should be 1.5 spaced. See the setstretch command shown above in the tex file.
+\subsection{Design Problem} % Ethan
+% Explain the design problem Hardware software timeline
+% 275 words max
+\subsection{System Specifications} % Ethan
+% 1-2 tables (maybe one for HW SW)
+% Ethan will write some hardware and software specs here in a table
 
-\section{Work Breakdown Structure} % Eric
-% A bit of preamble followed by WBS diagram figure
+\section{Work Breakdown Structure} % ERIC
+% 1.5 pages max for everything
+\subsection{Approach} 
+\subsection{Design Tools, Hardware, Instrumentation}
+\subsection{Validation}
 
 \section{Progress to Date} % Ethan / Tim for HW/SW respectively
+% Explain that there has been a completed hardware design.
+% include schematics, layout and BOM in Appendices
 \subsection{Hardware Design}
 \subsection{Software Design}
```

Comment:
Perhaps better to omit these subsections so that Milestones is numbered 3.1 as in the template

Reply:
Omit as in just keep them as separate paragraphs without headers

Reply:
So we're gonna keep these? Don't think there's going to be a lot of content to warrant adding two extra headers